README.md

```diff
@@ -73,10 +74,9 @@ Terraform creates a state file called tfstate and it can contain sensitive data
 
 ### Azure Remote Provider Shell (backends\azure_tf_backend)
 
-The Azure Remote Provider shell is used in order to enable CloudShell access to Azure storage to be used in order to 
-store the remote statefile.</br>
+The Azure Remote Provider shell is used in order to enable CloudShell access to Azure storage, to then be used to store the remote state file.</br>
 One must create a resource and fill in the attributes - then specify that resource name as the Remote State Provider.
-Only one type of authentication is allowed either by Access Key or using the Cloud Provider authentication keys.
+Only one type of authentication is allowed, either by Access Key or using the Cloud Provider authentication keys. ? if both are specified it errors?
```

Comment:
?if both are specified it errors? - correct. It will throw an error during resource Autoload

Reply:
Added text about error